src/components/pages/Employees/RenderEmployeesPage.js

```diff
@@ -1,9 +1,10 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
```

Comment:
Are you comfortable using scss instead of Styled-Components? If not, we can keep it, but this would be the only file that has Styled Components.